docs/gallery/tutorial/autogen/materials_science_ase.py

```diff
@@ -0,0 +1,316 @@
+"""
+=====================================================
+Computational materials science with ASE
+=====================================================
+
+Introduction
+============
+Unlock the power of automated and reproducible computational science!
+This tutorial will guide you through building, running, and visualizing computational workflows using `AiiDA-WorkGraph` and the `Atomistic Simulation Environment (ASE) <https://wiki.fysik.dtu.dk/ase/>`_.
+You'll learn how to construct complex pipelines where every calculation and data point is automatically tracked, ensuring your research is completely reproducible.
+
+We'll explore two key examples that highlight the flexibility of AiiDA-WorkGraph:
+
+1.  **Atomization energy**: A simple, linear workflow to calculate the atomization energy of a diatomic molecule.
+2.  **Equation of state (EOS)**: A more advanced workflow for bulk structure that showcases how to handle loops and dynamic inputs/outputs, a common pattern in dynamic workflows.
+
+"""
+
+# %%
+# First, ensure you have `aiida-workgraph`, `ase`, and a configured AiiDA environment.
+# If not, you can install the necessary packages:
+#
+# .. code-block:: console
+#
+#    pip install aiida-workgraph ase
+#
+# Then, load your AiiDA profile.
+
+from aiida import load_profile
+
+load_profile()
+
+# %%
+# Atomization energy of a diatomic molecule
+# ==========================================
+# The atomization energy (ΔE) is the energy required to break a molecule down into its individual, separate atoms.
+# For a diatomic molecule, the formula is straightforward:
+#
+# .. math::
+#
+#    \Delta E = 2 \times E_{\text{atom}} - E_{\text{molecule}}
+#
+# Where :math:`E_{\text{atom}}` is the total energy of a single atom and :math:`E_{\text{molecule}}` is the total energy of the molecule.
+#
+# To build our workflow, we'll start with standard Python functions and convert them into AiiDA-trackable components using the ``@task`` decorator.
+#
+# .. note::
+#
+#    We're using the ASE EMT (Effective Medium Theory) calculator because it's exceptionally fast and perfect for demonstrations.
+#    You can easily swap it with any other ASE-compatible calculator, like Quantum ESPRESSO, VASP, or GPAW, for your research.
+#
+from aiida_workgraph import task
+from ase import Atoms
+from ase.build import molecule
+
+
+@task()
```

Comment:
```suggestion
@task
```

In general (also in the docs), we should be consistent. If no arguments are passed, there is no need for `()`. This is common for decorator usage 🙂